ui-library-add-call-recording/api/stopRecording.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace Contoso
+{
+    public static class stopRecording

```

Comment:
Classes start with uppercase

![image](https://user-images.githubusercontent.com/79329532/151877712-0f4615e3-d40b-46ce-a87b-e3f83da9ce96.png)


https://www.c-sharpcorner.com/UploadFile/8a67c0/C-Sharp-coding-standards-and-naming-conventions/#:~:text=C%23%20Naming%20Conventions%20%20%20Abbreviations%20%20,%20%20DropDownList%20%2021%20more%20rows%20

Reply:
Thanks for that reference. This is the template generated by vscode. We do want the API endpoint name to be camelCase. I'll rename the classes in the next PR.